### PR TITLE
Puppet 4

### PR DESCRIPTION
**Pull request by pranav**
@kenbreeman @solomonty

### CODE REVIEW

File: templates/my.cnf.pass.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [client]
 user=root
 host=localhost
-<% unless root_password == 'UNSET' -%>
-password=<%= root_password %>
+<% unless @root_password == 'UNSET' -%>
+password=<%= @root_password %>
```
Comment (solomonty) [inline templates/my.cnf.pass.erb:5]: not `mysql::root_password` ?

Comment (pranav) [inline templates/my.cnf.pass.erb:5]: `@` is the correct sign for getting a variable from out of the scope of this file. I'm sending in another PR with more changes to the ERBs here

**Comment by solomonty** [comment]
current version of the file

https://github.com/puppetlabs/puppetlabs-mysql/blob/master/templates/my.cnf.pass.erb

**Comment by pranav** [comment]
We're on the 2013 version of this module. I'm all for upgrading to the latest version, the only diffs in this repo are the ones for puppet4 im adding.

**Comment by solomonty** [comment]
I showed you the current file because they use different syntax for accessing the variable.

**Pull request closed without merging**